src/main/java/trello/entities/List.java

```diff
@@ -12,6 +12,9 @@
 
 package trello.entities;
 
+import java.util.ArrayList;
+import java.util.Comparator;
+
```

Comment:
Please, change the class name of comment bellow this line.

Reply:
done